managed/models/database.go

```diff
@@ -1057,30 +1060,66 @@ func setupFixture1(q *reform.Querier, username, password string) error {
 		return err
 	}
 
+	address := strings.Split(params.Address, ":") // TODO: find better solution
```

Comment:
Maybe 

`host, port, err := net.SplitHostPort("hello.com:81")`

Reply:
in that case if we pass docker-compose like service name "some-server" without port we get an error that port is not provided

Reply:
split host port to parse address done

Reply:
So then maybe it can be:
If it contains : then return service name
otherwise net.SplitHostPort. 

If its still actual after changes and its up to you.

Reply:
proposal is implemented)